development/compiling/introduction_to_the_buildsystem.rst

```diff
@@ -124,63 +124,56 @@ you can build them yourself).
 Aside from that, there are a few standard options that can be set in all
 build targets, and which will be explained below.
 
-.. _doc_introduction_to_the_buildsystem_tools:
+.. _doc_introduction_to_the_buildsystem_target:
 
-Tools
------
+Target
+------
 
-Tools are enabled by default in all PC targets (Linux, Windows, macOS),
-disabled for everything else. Disabling tools produces a binary that can
-run projects but that does not include the editor or the project
-manager.
+Target controls if the editor is contained and used debug flags.
+All builds are optimized. Each mode means:
 
-::
+-  **editor**: Build with editor, optimized, with debugging code
+-  **template_debug**: Build with C++ debugging symbols
+-  **template_release**: Build without symbols
 
-    scons platform=<platform> tools=yes/no
+The editor is enabled by default in all PC targets (Linux, Windows, macOS),
+disabled for everything else. Disabling the editor produces a binary that can
+run projects but that does not include the editor or the project manager.
 
-.. _doc_introduction_to_the_buildsystem_target:
+::
 
-Target
-------
+    scons platform=<platform> target=editor/template_debug/template_release
 
-Target controls optimization and debug flags. Each mode means:
+Dev build
+---------
 
--  **debug**: Build with C++ debugging symbols, runtime checks (performs
```

Comment:
Shouldn't we still explain the different targets?
- editor
- template_debug
- template_release